src/app/overview/overview.component.ts

```diff
@@ -509,7 +524,14 @@ export class OverviewComponent extends SubscriptionManager implements OnInit {
         if (qpVals) {
           this.queryParamsRaw[facetName].forEach((val: string) => {
             if (!opsFromData.includes(val)) {
-              opsFromQP.push(this.formatNLV(prefix, val, false));
+              opsFromQP.push(
```

Comment:
Again, what is `opsFromQP`? Please avoid abbreviations like this, it's unclear what it is

Reply:
Renamed